iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -44,11 +44,11 @@ static llvm::cl::opt<int> matmulWorkgroupTileSize(
     llvm::cl::desc(
         "linalg.matmul tile size for workgroups spliting of M, N dimension"),
     llvm::cl::init(64));
-static llvm::cl::opt<int> matmulL1TileSize(
+static llvm::cl::list<int> clMatmulL1TileSize(
```

Comment:
Actually we can drop this. Dont think anyone should be relying on this, and is providing no value.